ark/segmentation/marker_quantification.py

```diff
@@ -21,21 +22,36 @@ def compute_marker_counts(input_images, segmentation_masks, nuclear_counts=False
             rows x columns x compartment matrix of masks
         nuclear_counts (bool):
             boolean flag to determine whether nuclear counts are returned
+        regionprops_features (list):
+            morphology features for regionprops to extract for each cell
 
     Returns:
         xarray.DataArray:
             xarray containing segmented data of cells x markers
     """
 
-    unique_cell_ids = np.unique(segmentation_masks[..., 0].values)
+    if regionprops_features is None:
+        regionprops_features = ['label', 'area', 'eccentricity', 'major_axis_length',
```

Comment:
Would a variable like this be a candidate for settings.py once someone addresses this issue?

Reply:
You see exactly what my thought-process was. Yes exactly!

Reply:
noted